### ISSUE TITLE
How to get output which is in same size as input image?

### ISSUE THREAD

**Issue by saicoco** [jakeret/tf_unet#138]
Thanks for your awesome tf_unet api, it helps me a lot.

A problem I wanna to ask: how to get the output that in same size with input. As I know, padding input can get the output， however， it will bring some noises into result.  How to do it, thanks!

**Comment by jakeret** [comment]
Great to hear that this is of use to you!
The way the network works you will always get a small sized image. 
If you look thru older issues people are usually mirroring the edges of the image to compensate for the loss. This gives generally good results

**Comment by AlibekJ** [comment]
Joel, seems like this question should go to somekind of FAQ. Where do you
think is the best place for it?

**Comment by saicoco** [comment]
Got it @jakeret

**Comment by jakeret** [comment]
@AlibekJ agreed, this seems to raise a lot of confusion. How about a new section (e.g. 'Data handling') in the documentation that would address this and other data related topics.
Didn't you have an implementation of this edge mirroring?

**Comment by saicoco** [comment]
@314rated I get the same shape with input by padding large value to input, and you can center crop the output in input_size when you get the output image.

**Comment by 314rated** [comment]
@saicoco Thanks! I shall try that
